clients/java/src/test/java/marquez/client/models/DbTableTest.java

```diff
@@ -17,16 +17,22 @@
 import static marquez.client.models.ModelGenerator.newDbTable;
 import static org.assertj.core.api.Assertions.assertThat;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
+import java.util.UUID;
+import marquez.client.Utils;
 import org.junit.jupiter.api.Test;
 
 @org.junit.jupiter.api.Tag("UnitTests")
 public class DbTableTest {
-  private static final Dataset DB_TABLE = newDbTable();
```

Comment:
I would add a helper method `newDbTableWith(UUID)`, where `UUID` is the current version you'd like to assert.